public/resources/js/alcom-versions-loader.js

```diff
@@ -0,0 +1,101 @@
+function detectPlatform() {
+    const platform = window.navigator?.userAgentData?.platform || window.navigator.platform || 'Unknown';
+    const lowerCasePlatfrom = platform.toLowerCase();
+
+    if (lowerCasePlatfrom.indexOf('mac') !== -1) {
+        return 'macOS';
+    } else if (lowerCasePlatfrom.indexOf('win') !== -1) {
+        return 'Windows';
+    } else if (/Linux/.test(platform)) {
+        return 'Linux';
+    } else if (lowerCasePlatfrom == 'android') {
+        return 'Android';
+    } else if (lowerCasePlatfrom == 'ios' || platform == 'ipad' || platform == 'ipod') {
+        return 'iOS';
+    } else {
+        console.log('Unsupported platform: ' + platform);
+        return 'Unsupported';
+    }
+}
+
+let _mainButtonLink;
+
+async function init() {
+    await waitUntilLocaleIsLoaded();
+
+    const errorNotification = document.getElementById('err-failedToLoadLatestVersion');
+    const loadingButton = document.getElementById('btn-download-loading');
+    const errorButton = document.getElementById('btn-download-failed');
+
+    try {
+        const response = await fetch('/api/gui/tauri-updater.json')
+        const data = await response.json()
+        const version = data.version;
+
+        const macLink = `https://github.com/vrc-get/vrc-get/releases/download/gui-v${version}/ALCOM-${version}-universal.dmg`;
+        const windowsLink = `https://github.com/vrc-get/vrc-get/releases/download/gui-v${version}/ALCOM-${version}-x86_64-setup.nsis.zip`;
+        const windowsStandaloneLink = `https://github.com/vrc-get/vrc-get/releases/download/gui-v${version}/ALCOM-${version}-x86_64.exe`;
+        const linuxLink = `https://github.com/vrc-get/vrc-get/releases/download/gui-v${version}/alcom-${version}-x86_64.AppImage`;
+
+        document.getElementById('btn-download-windowsSetup').href = windowsLink;
+        document.getElementById('btn-download-windowsStandalone').href = windowsStandaloneLink;
+        document.getElementById('btn-download-macOS').href = macLink;
+        document.getElementById('btn-download-linux').href = linuxLink;
```

Comment:
I think we can and it's better include latest version information in alcom/index.hbs by passing data from build.js.

Reply:
To embed latest ALCOM version, we need to pass data through handlebars so I think it's good to have `data` namespace instead of using `__something`.